### ISSUE TITLE
link services titles to docs

### ISSUE THREAD

**Issue by arlin** [phylotastic/phylotastic.github.io#7]
Currently, there is a "web services" page, with titled sections for each service-- but the titles do not link to web service docs or more detailed descriptions.  This is a mistake.  Users expect this page to link to detailed info about services.

**Comment by abusalehmdtayeen** [comment]
@arlin , where is this "web services" page? Is it in phylotastic.org website?

**Comment by arlin** [comment]
https://github.com/phylotastic/phylotastic.github.io/blob/master/services.html

**Comment by arlin** [comment]
yes, this is phylotastic.org

**Comment by arlin** [comment]
I'm sorry that this was not clear.  This has not been fixed.  It should only take 5 minutes to fix because "Documentation" is already linked to the correct target.  You just need to link the title to that same target.  I went to github to fix this myself, but I found that the website at phylotastic.org is ahead of the repository.  It doesn't have the "Usage details are in the Documentation" statements.  

So, step 1 is to check in your changes and push them to the main repository.  

Step 2 is to make sure that the *titles* are links to the docs.  Users will click on the titles and expect them to do something. 

<img width="809" alt="screen shot 2017-03-08 at 4 59 32 pm" src="https://cloud.githubusercontent.com/assets/1659198/23726222/849ae34e-0421-11e7-96e1-417e9db3bf1a.png">
